_haikus/Note_to_self.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: haiku
+title: Note to self
+author: Abubakar
+---
+
+
+In life being truthful
+to yourself requires patience
```

Comment:
This should have 7 syllables and should end with two spaces, you can try:

```
to you requires patience
```

Reply:
Thank you @ltagliaferri i , I have made a new commit